frontend/src/components/MainNavigation.vue

```diff
@@ -36,84 +36,99 @@ limitations under the License.
+          v-model="projectMenu"
+        >
+          <v-btn
+            block
+            slot="activator"
+            class="project-selector elevation-4 ma-0 white--text"
+            @keydown.down="selectProjectWithKeys('down')"
+            @keydown.up="selectProjectWithKeys('up')"
+            @keyup.enter="navigateToHighlightedProject"
+          >
+            <v-icon class="pr-4">mdi-grid-large</v-icon>
+            <span class="ml-2">{{projectName}}</span>
+            <v-spacer></v-spacer>
+            <v-icon right>{{projectMenuIcon}}</v-icon>
+          </v-btn>
+
+          <v-card light>
+            <template v-if="projectList.length > 3">
+              <v-card-title class="pa-0 grey lighten-5">
+                <v-text-field
+                  light
+                  clearable
+                  label="Filter projects"
+                  single-line
+                  hide-details
+                  full-width
+                  color="grey darken-1"
+                  prepend-icon="search"
+                  class="ml-4 project-filter"
+                  v-model="projectFilter"
+                  ref="projectFilter"
+                  @keyup.esc="projectFilter = ''"
+                  @keyup.enter="navigateToHighlightedProject"
+                  @input="onInputProjectFilter"
+                  @keydown.down="selectProjectWithKeys('down')"
+                  @keydown.up="selectProjectWithKeys('up')"
+                  autofocus
+                >
+                </v-text-field>
+              </v-card-title>
+              <v-divider></v-divider>
+            </template>
+            <v-list light class="project-list" ref="projectList" @scroll.native="handleProjectListScroll">
+              <v-list-tile
+                class="project-list-tile"
+                v-for="(project, index) in visibleProjectList"
+                @click="onProjectClick($event, project)"
+                :class="{'grey lighten-4' : index === highlightedProjectIndex}"
+                :key="index"
               >
-                <v-icon>add</v-icon>
-                <span class="ml-2">Create Project</span>
-              </v-btn>
-              <span>You are not authorized to create projects</span>
-            </v-tooltip>
-          </v-card-actions>
-        </v-card>
-      </v-menu>
+                <v-list-tile-avatar>
+                  <v-icon v-if="project.metadata.name === projectName" color="teal">check</v-icon>
+                </v-list-tile-avatar>
+                <v-list-tile-content>
+                  <v-list-tile-title>{{project.metadata.name}}</v-list-tile-title>
+                  <v-list-tile-sub-title class="project-owner">{{getProjectOwner(project)}}</v-list-tile-sub-title>
+                </v-list-tile-content>
+                <v-list-tile-action v-if="isProjectNameMatchingFilter(project.metadata.name)">
+                  <v-chip small color="cyan darken-2">Exact match</v-chip>
```

Comment:
Would not add the exact match chip.

Reply:
okay, we can remove it. I have no strong opinion about it. @petersutter WDYT?

Reply:
I thought it was a nice feature but I'm not attached to it so we can remove it

Reply:
removed